packages/mobile/src/components/account-balance/AccountBalance.tsx

```diff
@@ -0,0 +1,138 @@
+import { useMemo } from 'react'
+
+import { useCurrentUserId, useUserBalanceHistory } from '@audius/common/api'
+import { accountBalanceMessages as messages } from '@audius/common/messages'
+
+import {
+  Flex,
+  Text,
+  IconCaretUp,
+  IconCaretDown,
+  Paper
+} from '@audius/harmony-native'
+import LoadingSpinner from 'app/components/loading-spinner'
+import { UserBalanceHistoryGraph } from 'app/components/user-balance-history-graph'
+
+type AccountBalanceProps = {
+  userId?: number
+  width?: number
+  height?: number
+}
+
+const formatCurrency = (value: number): string => {
```

Comment:
think this and `formatCurrencyLarge` should be one function with a `decimals` arg. also, we have a `formatCurrency` in `decimals.ts`, can that help here?